.chainerci/Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM nvidia/cuda:10.0-devel
+
+LABEL author="Daisuke Tanaka <duaipp@gmail.com>, Shunta Saito <shunta.saito@gmail.com>"
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    ca-certificates \
+    build-essential \
+    curl \
+    wget \
+    git \
+    libssl-dev \
+    zlib1g-dev \
+    libbz2-dev \
+    libreadline-dev \
+    libsqlite3-dev \
+    libncurses5-dev \
+    libncursesw5-dev \
+    libffi-dev && \
+    rm -rf /var/lib/apt/lists/* /var/cache/apt/archives/*
+
+RUN curl -o cmake.tar.gz -LO https://github.com/Kitware/CMake/releases/download/v3.12.4/cmake-3.12.4-Linux-x86_64.tar.gz && \
```

Comment:
Do we need this? If yes, could you add a comment to explain why we need to install CMake from source instead of apt?

Reply:
Exactly! We can remove this, thx